domenetjenester/behandling-revurdering/src/main/java/no/nav/foreldrepenger/behandling/revurdering/ytelse/fp/UttakGrunnlagTjeneste.java

```diff
@@ -106,14 +107,38 @@ public ForeldrepengerGrunnlag grunnlag(BehandlingReferanse ref) {
             .medPleiepengerGrunnlag(pleiepengerGrunnlag(ref).orElse(null))
             .medOppdagetPleiepengerOverlappendeUtbetaling(behandling.harBehandlingÅrsak(BehandlingÅrsakType.RE_VEDTAK_PLEIEPENGER))
             .medUføretrygdGrunnlag(uføretrygdGrunnlag(ref).orElse(null))
-            .medNesteSakGrunnlag(nesteSakGrunnlag(ref).orElse(null));
+            .medNesteSakGrunnlag(nesteSakGrunnlag(ref).orElse(null))
+            .medDødsfall(harDødsfall(behandling, familiehendelser, ref))
+            ;
         if (fagsakRelasjon.isPresent()) {
             var annenpart = annenpart(fagsakRelasjon.get(), behandling);
             grunnlag = grunnlag.medAnnenpart(annenpart.orElse(null));
         }
         return grunnlag;
     }
 
+    private boolean harDødsfall(Behandling behandling, FamilieHendelser familiehendelser, BehandlingReferanse ref) {
+        var barna = familiehendelser.getGjeldendeFamilieHendelse().getBarna();
+
+        var harDødsdatoPåBarn = barna.stream().anyMatch(barn -> barn.getDødsdato().isPresent());
+        var harBehandlingÅrsakMedDød = behandling.getBehandlingÅrsaker()
+            .stream()
+            .anyMatch(årsak -> BehandlingÅrsakType.årsakerRelatertTilDød().contains(årsak.getBehandlingÅrsakType()));
+        return harDødsdatoPåBarn || harBehandlingÅrsakMedDød || erOpplysningerOmDødEndretIPersonopplysninger(ref);
+    }
+
+    private boolean erOpplysningerOmDødEndretIPersonopplysninger(BehandlingReferanse ref) {
+        var behandlingId = ref.behandlingId();
```

Comment:
Mener du å sjekke om en i persongalleriet har dødsdato (eller vil du se om dato er korrigert?)
PersonopplysningTjeneste  hentPersonopplysningerHvisEksisterer - lag en sjekkForNoenDøde() i PersonopplysningAggregat eller (helst ikke) getAktørPersonopplysningMap() - values og sjekk for dødsdato

Reply:
Vet ikke hvorfor man bare sjekker etter endringer her, kopiert fra tidligere. Men regner at det kan være noe rundt hvis søker/barn og til og fra status død så burde vi uansett fra AP i uttak. Men det er kanskje ikke så ofte at man først får status død som senere fjernes?

Reply:
Tror det er forsøk på å fange opp flest mulig saker som inkluderer dødsfall